sway-lib-core/src/ops.sw

```diff
@@ -1162,6 +1162,18 @@ impl Shift for u8 {
     }
 }
 
+impl u256 {
+    pub fn is_zero(self) -> bool {
+        self == 0x0000000000000000000000000000000000000000000000000000000000000000u256
+    }
+
+    pub fn is_even(self) -> bool {
+        let r = self % 0x0000000000000000000000000000000000000000000000000000000000000002u256;
+        r == 0x0000000000000000000000000000000000000000000000000000000000000000u256
```

Comment:
This is probably good enough for now, but I do wonder if `& 1` would cost more or less. Something to add to our optimizations when the instruction costs are properly tuned I guess.